tests/api_connexion/endpoints/test_user_endpoint.py

```diff
@@ -0,0 +1,225 @@
+                last_name=f'test{i}',
+                username=f'TEST_USER{i}',
+                email=f'mytest@test{i}.org',
+                roles=roles or [],
+                created_on=timezone.parse(DEFAULT_TIME),
+                changed_on=timezone.parse(DEFAULT_TIME),
+            )
+            for i in range(1, count + 1)
+        ]
+
+
+class TestGetUser(TestUserEndpoint):
+    def test_should_respond_200(self):
+        users = self._create_users(1)
+        self.session.add_all(users)
+        self.session.commit()
+        response = self.client.get("/api/v1/users/TEST_USER1", environ_overrides={'REMOTE_USER': "test"})
+        assert response.status_code == 200
+        assert response.json == {
+            'active': None,
+            'changed_on': DEFAULT_TIME,
+            'created_on': DEFAULT_TIME,
+            'email': 'mytest@test1.org',
+            'fail_login_count': None,
+            'first_name': 'test1',
+            'last_login': None,
+            'last_name': 'test1',
+            'login_count': None,
+            'roles': [],
+            'user_id': users[0].id,
+            'username': 'TEST_USER1',
+        }
+
+    def test_should_respond_404(self):
+        response = self.client.get("/api/v1/users/invalid-user", environ_overrides={'REMOTE_USER': "test"})
+        assert response.status_code == 404
+        assert {
+            'detail': "The User with username `invalid-user` was not found",
+            'status': 404,
+            'title': 'User not found',
+            'type': EXCEPTIONS_LINK_MAP[404],
+        } == response.json
+
+    def test_should_raises_401_unauthenticated(self):
+        response = self.client.get("/api/v1/users/TEST_USER1")
+        assert_401(response)
+
+    def test_should_raise_403_forbidden(self):
+        response = self.client.get(
+            "/api/v1/users/TEST_USER1", environ_overrides={'REMOTE_USER': "test_no_permissions"}
+        )
+        assert response.status_code == 403
+
+
+class TestGetUsers(TestUserEndpoint):
+    def test_should_response_200(self):
+        response = self.client.get("/api/v1/users", environ_overrides={'REMOTE_USER': "test"})
+        assert response.status_code == 200
+        assert response.json["total_entries"] == 2
+        usernames = [user["username"] for user in response.json["users"] if user]
+        assert usernames == ['test', 'test_no_permissions']
+
+    def test_should_raises_401_unauthenticated(self):
+        response = self.client.get("/api/v1/users")
+        assert_401(response)
+
+    def test_should_raise_403_forbidden(self):
+        response = self.client.get("/api/v1/users", environ_overrides={'REMOTE_USER': "test_no_permissions"})
+        assert response.status_code == 403
+
+
+class TestGetUsersPagination(TestUserEndpoint):
+    @parameterized.expand(
+        [
+            ("/api/v1/users?limit=1", ['test']),
+            ("/api/v1/users?limit=2", ['test', "test_no_permissions"]),
```

Comment:
Just for consistency

```suggestion
            ("/api/v1/users?limit=1", ["test"]),
            ("/api/v1/users?limit=2", ["test", "test_no_permissions"]),
```